data/remnant/remnant 1 introduction.txt

```diff
@@ -2435,6 +2435,60 @@ mission "Remnant: Salvage 6"
+			`	The Remnant steps forward and accepts the case from you. The case pops open, and he slots the crystal into a spot on the display. He nods as the display pops up several confirmations of data integrity. "Well, you have certainly made many ripples since you arrived in our little corner of the galaxy." He gestures and the large display zooms in on a spot to show you walking through a spaceport outfitter. The display shifts to show you helping wounded onto your ship. The scene shifts again to show you launching the small stealth surveillance satellite. Then the table shifts back to the display of troop movements.`
+			`	"You have not been with us too long, but you have shown yourself to be trustworthy and eager to help." He gestures at the display. "As you can see, your actions have not gone unobserved. We have decided to amend your IFF registration as a result. He points to a set of slots next to the table similar to the ones that are installed on your ship. "Please place both your licenses in those slots, and I will confirm your side of the change."`
+			choice
+				`	(place your licenses in the designated spots.)`
+					goto comply
+				`	"You have been watching me?"`
+			`	"Yes. There is very little that goes on in our worlds that is not observed by someone, and sometimes it garners more attention than others." He looks at you. "You are one such. As a stranger among us, you were subject to much attention, although you probably did not notice most of it. Some felt it was too risky to trust you as far as we did, but the consensus was that you are worth the trust. Now," he gestures towards the console again. "place your licenses there please."`
+			label comply
+			`	He slots several things that look similar to but different from your licenses into slots on his side of the table and taps several controls. "You are not yet one of us, but this is one step closer to being such. Your IFF signature has been updated, and our ship's databases will be updated within a day." He removes the licenses and hands yours back to you with a relieved expression. "We are a curious people, and we fully understand the search for knowledge and to learn more about one's surroundings. But we also know how important it is to maintain our secrets.`
+			`	"Going forward, you are now authorized to scan Remnant ships, and we encourage you to do so, although you should wait a day or two for the update to propagate through our fleet." He looks amused by something, but his serious expression returns. "Scanning each other is, after all, a valuable form of security that gives us opportunities to spot infiltrators and unknown contaminants on returning ships. Scans are uploaded to our databases for signature matching via the Quantum Entangled Communicator built into the IFF mechanism onboard your ship, and we will return information such as alien outfit identification and tactical scan readouts whenever possible to help you."`
```

Comment:
How does this thing about them sending information to me work?

Reply:
You already see some of it. This is basically an in-game explanation for how you can head off and scan a ship from some alien race you've never met before and get consistent names for the outfits as well as a decent idea of what they do. Namely, that scan is being fed through the QEC (or at least, a hash of it that can be matched against records, since the actual scan is probably a fairly big data file), and they can either return a name and some info on it; or they can compare to their records and provide some feedback as to what it might be.

Granted, this doesn't solve the fact that we can do the same thing with a regular human scanner on a human ship from day 1; but we can imagine that a human scanner probably has records of all the typical human equipment. So while this doesn't actually fill in the plot hole, it at least provides a reasonable explanation for how this manages to work for the Remnant and the player working with them.